src/CollectWidget.res

```diff
@@ -43,6 +78,12 @@ let make = (
     setFieldValidityDict(_ => Dict.make())
   }
 
+  // Reset form on PMT updation
+  React.useEffect(() => {
+    resetForm()
```

Comment:
why not move this as well to the previous useEffect if the dependency is the same (line 71)

Reply:
It's a different dependency - `selectedPaymentMethod` vs `selectedPaymentMethodType`